jsconfig.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
Perhaps replace this with a tsconfig?

Reply:
We have a TSconfig already. This is just old junk

Reply:
ah yes, my bad